fabfile/run.py

```diff
@@ -203,3 +202,32 @@ def tpch_queries(query_info, connectionURI, pg_version, citus_version, config_fi
         out_val = run(run_string)
         results_file.write(out_val)
         results_file.write('\n')
+
+@task
+@roles('master')
+def valgrind(*args): 
+    'Runs valgrind tests'
+
+    # validate arguments
+    if len(args) != 2 or args[0] not in ('enterprise', 'citus'):
+        abort('You must provide two arguments, with a command such as "run.valgrind:citus,v6.0.1".\n\
+            First argument can only be "citus" or "enterprise".')
+
+    citus_repo = args[0]
+    citus_version = args[1]
+
+    # install citus and set citus path variable
+    if citus_repo == 'citus':
+        repo_path = config.CITUS_REPO
+    else:
+        repo_path = config.ENTERPRISE_REPO
+    
+    setup.valgrind(citus_repo, citus_version)
+
+    with cd(os.path.join(repo_path, config.RELATIVE_REGRESS_PATH)):
+        run('make check-multi-vg VALGRIND_LOG_FILE=logs.txt || true')
```

Comment:
Can you create a variable in `config.py` and maybe use something like `valgrind_logs.txt` otherwise it is not clear. 

What is the reason for using `` || true`` if the run of valgrind test fails, do we still want to continue? If so why?

Reply:
> Can you create a variable in config.py and maybe use something like valgrind_logs.txt otherwise it is not clear.

That makes sense

> What is the reason for using || true if the run of valgrind test fails, do we still want to continue? If so why?

This is just because I migrated the existing linux commands from tools repository :). But it does not makes sense to me as well, I will remove `|| true`